### PR TITLE
feat: add importer for French cadastre

### PR DESCRIPTION
**Pull request by yohanboniface**
Screenshot From 2024-10-18 12-48-23

### CODE REVIEW

**Comment by yohanboniface** [comment]
@Aurelie-Jallut do you want to create a dedicated icon ?

**Comment by Aurelie-Jallut** [comment]
let's try !
image

image

File: umap/static/umap/js/modules/importers/cadastrefr.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import { DomUtil } from '../../../vendors/leaflet/leaflet-src.esm.js'
+import { BaseAjax, SingleMixin } from '../autocomplete.js'
+import * as Util from '../utils.js'
+import { AutocompleteCommunes } from './communesfr.js'
+
+const TEMPLATE = `
+  <h3>Cadastre</h3>
+  <p>Importer le cadastre d'une commune française..</p>
```
Comment (davidbgk) [inline umap/static/umap/js/modules/importers/cadastrefr.js:8]:
```suggestion
  <p>Importer le cadastre d’une commune française…</p>
```

File: umap/static/umap/js/modules/importers/cadastrefr.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  <select name="theme">
+    <option value="batiments">Bâtiments</option>
+    <option value="communes">Communes</option>
+    <option value="feuilles">Feuilles</option>
+    <option value="lieux_dits">Lieux dits</option>
+    <option value="parcelles" selected>Parcelles</option>
```
Comment (davidbgk) [inline umap/static/umap/js/modules/importers/cadastrefr.js:14]: I'm tempted to put the selected choice at the top of the list, it is a more classic pattern (and I get that it's sorted alphabetically for now but still).

**Review by davidbgk — APPROVED** [review]
So good 🙌

File: umap/static/umap/js/modules/importers/cadastrefr.js (hunk 2 above)
Comment (yohanboniface) [reply umap/static/umap/js/modules/importers/cadastrefr.js:14]: I'm tempted to think that given the list is small, the alphabetical order is more natural and convenient once open. Let's try that way and see ?

**Pull request merged**